scalardb-test/build.gradle

```
@@ -35,6 +42,8 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-dbcp2', version: '2.8.0'
     implementation group: 'com.scalar-labs', name: 'scalardb-sql', version: '4.0.0-SNAPSHOT'
     implementation group: 'com.scalar-labs', name: 'scalardb-sql-jdbc', version: '4.0.0-SNAPSHOT'
+    implementation group: 'com.scalar-labs', name: 'scalardb-graphql', version: '4.0.0-SNAPSHOT'
+    implementation 'com.graphql-java:graphql-java:17.2'
```

```suggestion
    implementation group: 'com.graphql-java', name: 'graphql-java', version: '17.2'
```
It's a minor detail, but it is to stay consistent with the other dependencies' import syntax.

---

@ymorimo Are you going to modify it?

---

Also, I think we should update it to `17.4` since the following PR have updated it:
https://github.com/scalar-labs/scalardb-graphql/pull/49

---

Updated the build.gradle. Thank you!